ccxt.d.ts

```diff
@@ -220,6 +220,54 @@ declare module 'ccxt' {
         // allow dynamic keys
         [key: string]: any;
         // properties
+        version: string;
+        apiKey: string;
+        secret: string;
+        password: string;
+        uid: string;
+        requiredCredentials: {
+            apiKey: boolean;
+            secret: boolean;
+            uid: boolean;
+            login: boolean;
+            password: boolean;
+            twofa: boolean;
+            privateKey: boolean;
+            walletAddress: boolean;
+            token: boolean;
+        };
+        options: {
+            [key: string]: any;
+            fetchTradesMethod: 'publicGetAggTrades' | string;
+            fetchTickersMethod: 'publicGetTicker24hr' | string;
+            defaultTimeInForce: 'GTC' | string;
+            defaultLimitOrderType: 'limit' | 'market' | string;
+            hasAlreadyAuthenticatedSuccessfully: boolean;
+            warnOnFetchOpenOrdersWithoutSymbol: boolean;
+            recvWindow: number;
+            timeDifference: number;
+            adjustForTimeDifference: boolean;
+            parseOrderToPrecision: boolean;
+            newOrderRespType: {
+                market: 'FULL' | string;
+                limit: 'RESULT' | string;
+            };
+        };
+        urls: {
+            logo: string;
+            api: {
```

Comment:
Is this correct? Looking at coinbasepro.js it's a single URL. Also, https://github.com/ccxt/ccxt/wiki/Manual says:

> urls['api']: The single string literal base URL for API calls or an associative array of separate URLs for private and public APIs.

Reply:
Fixed.